client/app/components/project/package-list-item.hbs

```diff
@@ -1,35 +1,33 @@
-<ol class="no-bullet">
-  <li>
-    {{#if (eq @package.statuscode (optionset 'package' 'status' 'code' 'REVIEWED_REVISIONS_REQUIRED'))}}
-      <FaIcon @icon="exclamation-triangle" class="text-gold" />
-    {{else}}
-      <FaIcon
-        @icon={{if (eq @package.statuscode (optionset 'package' 'status' 'code' 'PACKAGE_PREPARATION')) 'edit' 'check'}}
-        class={{if (eq @package.statuscode (optionset 'package' 'status' 'code' 'PACKAGE_PREPARATION')) '' 'text-green-dark'}}
-      />
-    {{/if}}
-    {{#if (eq @package.dcpPackagetype (optionset 'package' 'type' 'code' "PAS_PACKAGE"))}}
-      <LinkTo
-        @route={{if (eq @package.statuscode (optionset 'package' 'status' 'code' 'PACKAGE_PREPARATION')) 'pas-form.edit' 'pas-form.show'}}
-        @model={{@package.id}}
-        data-test-package-link={{@package.id}}
-      >
-        <strong>Version {{@package.dcpPackageversion}}</strong>
-      </LinkTo>
-    {{/if}}
-    {{#if (eq @package.dcpPackagetype (optionset 'package' 'type' 'code' 'RWCDS'))}}
-      <LinkTo
-        @route={{if (eq @package.statuscode (optionset 'package' 'status' 'code' 'PACKAGE_PREPARATION')) 'rwcds-form.edit' 'rwcds-form.show'}}
-        @model={{@package.id}}
-        data-test-package-link={{@package.id}}
-      >
-        <strong>Version {{@package.dcpPackageversion}}</strong>
-      </LinkTo>
-    {{/if}}
-    <span class="text-gray">| {{if (eq @package.statuscode (optionset 'package' 'status' 'code' 'PACKAGE_PREPARATION')) 'Editable since ' 'Submitted '}}
-      <Ui::DateDisplay @date={{@package.dcpStatusdate}} @outputFormat="MM/D/YYYY" @errorMessage="Date Unknown"/>
-    </span>
-  </li>
-</ol>
+<li>
+  {{#if (eq @package.statuscode (optionset 'package' 'status' 'code' 'REVIEWED_REVISIONS_REQUIRED'))}}
```

Comment:
Yay, optionsets! :) Just noting here that in my incoming PR, I am changing the `status` key to `statuscode`, since `statuscode` is actually the field name on the Package model. Just thought it would be more predictable, and developers wouldn't need to learn the idiosyncratic field-key mappings in the optionset helper.

Reply:
Either one of these two PRs will have to be updated depending on which is merged first.

Reply:
Oh wait, these are just indentation changes !

Reply:
Yeah, just whitespace. Hopefully git can sort that out.